public/app/features/alerting/unified/components/export/providers.ts

```diff
@@ -32,12 +43,51 @@ export const HclExportProvider: ExportProvider<'hcl'> = {
   type: 'terraform',
 };
 
+export const OperatorExportProvider: ExportProvider<'operator'> = {
+  name: 'Grafana Operator',
+  exportFormat: 'operator',
+  overrideLanguage: 'yaml',
+  type: 'operator',
+  supports: ['AlertRuleGroup', 'AlertRule', 'Receiver', 'RuleFolder'],
+  formatter: (raw: string) => {
+    const out: any[] = [];
+    const parsed = loadAll(raw) as Array<{ kind: string }>;
+    for (const resource of parsed) {
+      console.log(resource);
+      if (resource.kind === 'GrafanaAlertRuleGroup') {
+        const r = resource as any;
+        out.push({
+          ...r,
+          spec: {
+            ...r.spec,
+            rules: r.spec.rules.map(({ notification_settings, ...rule }: any) => ({
+              ...rule,
+              for: rule.for.endsWith('m') ? rule.for + '0s' : rule.for,
```

Comment:
This prevents the change noticed in https://github.com/grafana/grafana-operator/issues/1492